CFC_DataCollector/recommender/common.py

```diff
@@ -1,75 +1,108 @@
 from __future__ import division
 import urllib2
-import xml.etree.ElementTree as ET
 from get_database import *
 from datetime import datetime, timedelta
-import random
 import json
-import jsonpickle
-from trip import E_Mission_Trip
+from trip import *
+import copy
+import uuid
+import xml.etree.ElementTree as ET
+
+DATE_FORMAT = "%Y%m%dT%H%M%S-%W00" #This is a great hack thought of by Shaun
+CLASS_UUIDS = {
```

Comment:
!!!
Why don't we just put all our UUIDs on a giant screen in the middle of the internet for everybody to look at
!!!